m4_sessions/1806-inning/group8.md

```diff
@@ -1,22 +1,32 @@
 ## Group Members:
-
+Margaret, Clee, Camilo, Daniela
 
 
 ## Name of Session: 
+All These Flavors and You Chose to be Salty
 
 ## Session Description 
-
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+The benefits of being positive 
```

Comment:
Get a little more specific--what are the benefits?  How can it help you get a job?  Or create better experience?  Or you could say 100% of people who are positive get jobs, or something like that to catch excitement